### ISSUE TITLE
An error occurred while Layout/EmptyLinesAroundArguments cop was inspecting...

### ISSUE THREAD

**Issue by pdobb** [rubocop-hq/rubocop#9141]
After upgrading from rubocop 1.4.2 to 1.5.0 I'm seeing errors coming from the Layout/EmptyLinesAroundArguments cop when the method being called isn't explicitly named. e.g. an explicit `.call()` works fine but the implicit form of that, with `.()` does not.

Quick overview of the error message:

> An error occurred while Layout/EmptyLinesAroundArguments cop was inspecting ...
undefined method `line' for nil:NilClass

The line of code at fault appears to be https://github.com/koic/rubocop/commit/43daf1ede153d333a5cfac40e278b1cdd7206eaf#diff-5b6ef42034752294d379ffc00ed78e37dd4853da3e6700ac1250ec44a81c1aeeR49

--------

## Expected behavior

No error should occur in either case:

```ruby
# Works fine
MyServiceObject.call(
  arg1)

# Works fine
MyServiceObject.(arg1)

# Causes an error
MyServiceObject.(
  arg1)
```

## Actual behavior

```bash
$ rubocop --debug --only Layout/EmptyLinesAroundArguments rcs/temp.rb
For /Users/pauld/dev/badger: configuration from /Users/pauld/dev/badger/.rubocop.yml
configuration from /Users/pauld/.gem/ruby/2.6.6/gems/rubocop-performance-1.9.0/config/default.yml
configuration from /Users/pauld/.gem/ruby/2.6.6/gems/rubocop-performance-1.9.0/config/default.yml
Default configuration from /Users/pauld/.gem/ruby/2.6.6/gems/rubocop-1.5.0/config/default.yml
configuration from /Users/pauld/.gem/ruby/2.6.6/gems/rubocop-rails-2.8.1/config/default.yml
configuration from /Users/pauld/.gem/ruby/2.6.6/gems/rubocop-rails-2.8.1/config/default.yml
configuration from /Users/pauld/.gem/ruby/2.6.6/gems/rubocop-rake-0.5.1/config/default.yml
configuration from /Users/pauld/.gem/ruby/2.6.6/gems/rubocop-rake-0.5.1/config/default.yml
Inspecting 1 file
Scanning /Users/pauld/dev/badger/rcs/temp.rb
An error occurred while Layout/EmptyLinesAroundArguments cop was inspecting /Users/pauld/dev/badger/rcs/temp.rb:3:0.
undefined method `line' for nil:NilClass
/Users/pauld/.gem/ruby/2.6.6/gems/rubocop-1.5.0/lib/rubocop/cop/layout/empty_lines_around_arguments.rb:62:in `receiver_and_method_call_on_different_lines?'
/Users/pauld/.gem/ruby/2.6.6/gems/rubocop-1.5.0/lib/rubocop/cop/layout/empty_lines_around_arguments.rb:49:in `on_send'
/Users/pauld/.gem/ruby/2.6.6/gems/rubocop-1.5.0/lib/rubocop/cop/commissioner.rb:100:in `public_send'
/Users/pauld/.gem/ruby/2.6.6/gems/rubocop-1.5.0/lib/rubocop/cop/commissioner.rb:100:in `block (2 levels) in trigger_responding_cops'
/Users/pauld/.gem/ruby/2.6.6/gems/rubocop-1.5.0/lib/rubocop/cop/commissioner.rb:166:in `with_cop_error_handling'
/Users/pauld/.gem/ruby/2.6.6/gems/rubocop-1.5.0/lib/rubocop/cop/commissioner.rb:99:in `block in trigger_responding_cops'
/Users/pauld/.gem/ruby/2.6.6/gems/rubocop-1.5.0/lib/rubocop/cop/commissioner.rb:98:in `each'
/Users/pauld/.gem/ruby/2.6.6/gems/rubocop-1.5.0/lib/rubocop/cop/commissioner.rb:98:in `trigger_responding_cops'
/Users/pauld/.gem/ruby/2.6.6/gems/rubocop-1.5.0/lib/rubocop/cop/commissioner.rb:69:in `on_send'
/Users/pauld/.gem/ruby/2.6.6/gems/rubocop-ast-1.3.0/lib/rubocop/ast/traversal.rb:20:in `walk'
/Users/pauld/.gem/ruby/2.6.6/gems/rubocop-1.5.0/lib/rubocop/cop/commissioner.rb:86:in `investigate'
/Users/pauld/.gem/ruby/2.6.6/gems/rubocop-1.5.0/lib/rubocop/cop/team.rb:157:in `investigate_partial'
/Users/pauld/.gem/ruby/2.6.6/gems/rubocop-1.5.0/lib/rubocop/cop/team.rb:83:in `investigate'
/Users/pauld/.gem/ruby/2.6.6/gems/rubocop-1.5.0/lib/rubocop/runner.rb:315:in `inspect_file'
/Users/pauld/.gem/ruby/2.6.6/gems/rubocop-1.5.0/lib/rubocop/runner.rb:259:in `block in do_inspection_loop'
/Users/pauld/.gem/ruby/2.6.6/gems/rubocop-1.5.0/lib/rubocop/runner.rb:293:in `block in iterate_until_no_changes'
/Users/pauld/.gem/ruby/2.6.6/gems/rubocop-1.5.0/lib/rubocop/runner.rb:286:in `loop'
/Users/pauld/.gem/ruby/2.6.6/gems/rubocop-1.5.0/lib/rubocop/runner.rb:286:in `iterate_until_no_changes'
/Users/pauld/.gem/ruby/2.6.6/gems/rubocop-1.5.0/lib/rubocop/runner.rb:255:in `do_inspection_loop'
/Users/pauld/.gem/ruby/2.6.6/gems/rubocop-1.5.0/lib/rubocop/runner.rb:132:in `block in file_offenses'
/Users/pauld/.gem/ruby/2.6.6/gems/rubocop-1.5.0/lib/rubocop/runner.rb:157:in `file_offense_cache'
/Users/pauld/.gem/ruby/2.6.6/gems/rubocop-1.5.0/lib/rubocop/runner.rb:131:in `file_offenses'
/Users/pauld/.gem/ruby/2.6.6/gems/rubocop-1.5.0/lib/rubocop/runner.rb:122:in `process_file'
/Users/pauld/.gem/ruby/2.6.6/gems/rubocop-1.5.0/lib/rubocop/runner.rb:101:in `block in each_inspected_file'
/Users/pauld/.gem/ruby/2.6.6/gems/rubocop-1.5.0/lib/rubocop/runner.rb:100:in `each'
/Users/pauld/.gem/ruby/2.6.6/gems/rubocop-1.5.0/lib/rubocop/runner.rb:100:in `reduce'
/Users/pauld/.gem/ruby/2.6.6/gems/rubocop-1.5.0/lib/rubocop/runner.rb:100:in `each_inspected_file'
/Users/pauld/.gem/ruby/2.6.6/gems/rubocop-1.5.0/lib/rubocop/runner.rb:86:in `inspect_files'
/Users/pauld/.gem/ruby/2.6.6/gems/rubocop-1.5.0/lib/rubocop/runner.rb:47:in `run'
/Users/pauld/.gem/ruby/2.6.6/gems/rubocop-1.5.0/lib/rubocop/cli/command/execute_runner.rb:26:in `block in execute_runner'
/Users/pauld/.gem/ruby/2.6.6/gems/rubocop-1.5.0/lib/rubocop/cli/command/execute_runner.rb:52:in `with_redirect'
/Users/pauld/.gem/ruby/2.6.6/gems/rubocop-1.5.0/lib/rubocop/cli/command/execute_runner.rb:25:in `execute_runner'
/Users/pauld/.gem/ruby/2.6.6/gems/rubocop-1.5.0/lib/rubocop/cli/command/execute_runner.rb:17:in `run'
/Users/pauld/.gem/ruby/2.6.6/gems/rubocop-1.5.0/lib/rubocop/cli/command.rb:11:in `run'
/Users/pauld/.gem/ruby/2.6.6/gems/rubocop-1.5.0/lib/rubocop/cli/environment.rb:18:in `run'
/Users/pauld/.gem/ruby/2.6.6/gems/rubocop-1.5.0/lib/rubocop/cli.rb:65:in `run_command'
/Users/pauld/.gem/ruby/2.6.6/gems/rubocop-1.5.0/lib/rubocop/cli.rb:72:in `execute_runners'
/Users/pauld/.gem/ruby/2.6.6/gems/rubocop-1.5.0/lib/rubocop/cli.rb:41:in `run'
/Users/pauld/.gem/ruby/2.6.6/gems/rubocop-1.5.0/exe/rubocop:13:in `block in <top (required)>'
/Users/pauld/.rubies/ruby-2.6.6/lib/ruby/2.6.0/benchmark.rb:308:in `realtime'
/Users/pauld/.gem/ruby/2.6.6/gems/rubocop-1.5.0/exe/rubocop:12:in `<top (required)>'
/Users/pauld/.gem/ruby/2.6.6/bin/rubocop:23:in `load'
/Users/pauld/.gem/ruby/2.6.6/bin/rubocop:23:in `<main>'
.

1 file inspected, no offenses detected

1 error occurred:
An error occurred while Layout/EmptyLinesAroundArguments cop was inspecting /Users/pauld/dev/badger/rcs/temp.rb:3:0.
Errors are usually caused by RuboCop bugs.
Please, report your problems to RuboCop's issue tracker.
https://github.com/rubocop-hq/rubocop/issues

Mention the following information in the issue report:
1.5.0 (using Parser 2.7.2.0, rubocop-ast 1.3.0, running on ruby 2.6.6 x86_64-darwin19)
Finished in 0.43348300000070594 seconds
```

## Steps to reproduce the problem

Add the following to an otherwise empty ruby file:

```ruby
MyServiceObject.(
  arg1)
```

And then run rubocop with the Layout/EmptyLinesAroundArguments enabled:
`rubocop --debug --only Layout/EmptyLinesAroundArguments rcs/temp.rb`

## RuboCop version

```
$ rubocop -V
1.5.0 (using Parser 2.7.2.0, rubocop-ast 1.3.0, running on ruby 2.6.6 x86_64-darwin19)
  - rubocop-performance 1.9.0
  - rubocop-rails 2.8.1
  - rubocop-rake 0.5.1
```

**Comment by pdobb** [comment]
Whoops. This is a duplicate of https://github.com/rubocop-hq/rubocop/issues/9140

**Comment by koic** [comment]
Yeah, this issue is a dup with #9140 and will be fixed by #9142. Thank you for the feedback!
